examples/crud-negative-targeting.ts

```diff
@@ -0,0 +1,74 @@
+import { amazonAdvertising } from './auth'
```

Comment:
What happened with this file? Why are there these big changes in this PR? Seems unrelated?

Reply:
New `typedocs` version detected errors in our `examples`.
Then, I found that I forgot to update examples when I refactor `SponsoredBrandsProductTargetingOperation` (in `68d8cd4` commit).

Previous versions didn't found them.